spec/sport_ngin_aws_auditor/rds_instance_spec.rb

```diff
@@ -66,38 +66,40 @@ module SportNginAwsAuditor
                                                                  multi_az: false,
                                                                  db_instance_class: "db.t2.small",
                                                                  state: "active",
-                                                                 product_description: "mysql",
+                                                                 product_description: "oracle-se2 (byol)",
+                                                                 db_instance_count: 1,
                                                                  class: "Aws::RDS::Types::ReservedDBInstance")
         reserved_rds_instance2 = double('reserved_rds_instance', reserved_db_instances_offering_id: "555te4yy-1234-555c-5678-thisisafake!!",
                                                                  multi_az: false,
                                                                  db_instance_class: "db.m3.large",
                                                                  state: "active",
                                                                  product_description: "postgresql",
+                                                                 db_instance_count: 2,
                                                                  class: "Aws::RDS::Types::ReservedDBInstance")
         reserved_db_instances = double('db_instances', reserved_db_instances: [reserved_rds_instance1, reserved_rds_instance2])
         rds_client = double('rds_client', describe_reserved_db_instances: reserved_db_instances)
         allow(RDSInstance).to receive(:rds).and_return(rds_client)
       end
 
       it "should make a reserved_rds_instance for each instance" do
-        reserved_instances = RDSInstance::get_reserved_instances
+        reserved_instances = RDSInstance.get_reserved_instances
```

Comment:
Let's also go through the ec2_instance_spec.rb and the cache_instance_spec.rb and always call this method by using `.` not `::`. Even though either would work.... I like consistency in my code.